sw/device/silicon_creator/rom_ext/meson.build

```diff
@@ -51,6 +51,7 @@ test('sw_silicon_creator_rom_ext_boot_policy_unittest', executable(
 
 rom_ext_linkfile_slot_a = files(['rom_ext_slot_a.ld'])
 rom_ext_linkfile_slot_b = files(['rom_ext_slot_b.ld'])
+rom_ext_linkfile_pic = files(['rom_ext_va.ld'])
```

Comment:
I think the file name needs to be updated.

Reply:
fixed